modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -225,6 +225,30 @@ This ID can be chosen from the config paramater section `wnd.kerberos.servers` a
 This is the keytab file that has been described in xref:enterprise/authentication/kerberos.adoc#keytab-files[Keytab Files].
 * `ocservice` +
 This is the SPN of the service user that has been described in xref:enterprise/authentication/kerberos.adoc#service-principal-name-spn[Service Principal Name (SPN)].
+* `usermapping` +
+Define a usermapping if necessary or required.
+
+{empty}
+
+The usermapping key::
++
+Note that `usermapping` is only possible with the enabled and configured {oc-marketplace-url}/apps/user_ldap[LDAP Integration] app. WND will throw an error if not.
++
+As an example, when the UID that was pulled into the ownCloud database on the users first time login after the LDAP app has been configured, this UID will not get updated on the ownCloud side if it gets updated on the LDAP side. Such a change prevents a user to login via Kerberos via ownCloud. This change can happen when the UID was e.g. an eMail address and then changed to a user name. For such a case, a mapping can be defined to use a different LDAP attribute for the users UID. For the LDAP settings in ownCloud, go to menu:Settings[General > User Authentication > Advanced > Exposed User Attributes] and enter LDAP attributes to map. With the WND config key type `EALdapAttr`, you can assign one attribute per Kerberos Server ID for the mapping.
```

Comment:
>  this UID will not get updated on the ownCloud side if it gets updated on the LDAP side.

It seems to say that is possible to change the uid... Changing the uid isn't supported and will break ownCloud.
To clarify, it's your responsibility to ensure the value of the uid doesn't change neither in ownCloud nor in LDAP. If you choose to use the mail as uid, you must ensure that the mail doesn't change.

Reply:
fixed